interface-definitions/system-logs.xml.in

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <node name="logs" owner="${vyos_conf_scripts_dir}/system-logs.py">
+        <properties>
+          <help>Logging options</help>
+          <priority>9999</priority>
+        </properties>
+        <children>
+          <node name="logrotate">
+            <properties>
+              <help>Logrotate options</help>
+            </properties>
+            <children>
+              <node name="atop">
+                <properties>
+                  <help>Atop logs options</help>
+                </properties>
+                <children>
+                  <leafNode name="maxsize">
```

Comment:
Please use `max-size` to match other CLI nodes that already exist using this name.

Reply:
This is a controversial question. We are controlling here the `logrotate` option with the same name. As for me, I see here no relation to the rest places in CLI, but a clear interface for people who are already familiar with `logrotate`.
In general, sometimes renaming options in our own style is a great idea - for example when we integrate new tools into existing services with already established terminology. I just think that this is not one of such cases.

Reply:
It's only a hyphen which makes it easier readable on the CLI ;)

Reply:
Replaced to `max-size`.